security/KeePassXC/Portfile

```diff
@@ -123,3 +124,4 @@ post-destroot {
 
 test.run        yes
 test.target     test ARGS="-V"
+test.env        TMPDIR=/tmp
```

Comment:
I thought MacPorts base already set the TMPDIR environment variable.

Reply:
the previous TMPDIR was `{$workdir}/...` and the name was so long, that the `TestSSHAgent` test failed because of an `too long for Unix domain socket` error.